pkgs/development/python-modules/mergedb/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, pkgs, fetchPypi, setuptools, pyyaml, colorama, jinja2, setuptools-scm, git, python3Packages }:
+
+let
+  pname = "mergedb";
+  version = "0.1.1";
+in
+python3Packages.buildPythonPackage {
+  inherit pname version;
```

I suggest to disable the module on all releases which are EOL.

```suggestion
buildPythonPackage rec {
  pname = "mergedb";
  version = "0.1.1";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```